Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -882,6 +974,13 @@ def update_user_iam(default_base_dn, args, create_if_not_exists, mapper_out, dis
             if not ou_modified_succeed:
                 fail_to_modify.append("ou")
 
+            if manager_email := ad_user.get('manageremail'):
+                manager_dn = get_user_dn_by_email(manager_email)
+                modification = {'manager': [('MODIFY_REPLACE', manager_dn)]}
```

Comment:
do we need the `manager` field in the outgoing mapper if we override its modification (line 969) with this?

Reply:
you are correct I'll remove it